### PR TITLE
Add button for Id editor #188

### PR DESCRIPTION
**Pull request by anushasingh**
Fixes issue #188 

  -Added dropdown menu which has two options : Open in ID and Open in JOSM

### CODE REVIEW

File: public_src/components/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -67,11 +67,33 @@
...
-    <i class="fa fa-download" aria-hidden="true"></i>
-    &nbsp;{{ 'Open in JOSM' | translate }}
-  </button>
+
+  <div id="dropdown" class="btn-group col-md-6" dropdown>
+
+     <button dropdownToggle type="button" class="btn btn-default btn-block dropdown-toggle" container="body"
+             tooltip="{{ 'Open in JOSM/ID' | translate }}"
+             [disabled]="isDisabled()">
+            &nbsp;{{ 'Open in ..' | translate }}
```
Comment (dkocich) [inline public_src/components/toolbar/toolbar.component.html]: Please use "Open in ..." (with three dots) and add/remove texts from language files https://github.com/dkocich/osm-pt-ngx-leaflet/blob/develop/public_src/assets/i18n/en.json (keep it sorted alphabetically and keep string as a value when you do not know the translation - we will translate it later in POEditor.com)

File: public_src/components/toolbar/toolbar.component.ts [hunk 2]
```diff
@@ -170,6 +170,21 @@ export class ToolbarComponent {
...
+    {
+     zoomlevel=13;
+    }
+    const idHref = 
+    'https://www.openstreetmap.org/edit?editor=id#map='+
+    zoomlevel+'/'+
+    selection['lat']+'/'+
+    selection['lon'];   
+    window.open(idHref);
+  }
```
Comment (dkocich) [inline public_src/components/toolbar/toolbar.component.ts]: Please enable linter and use consistent style (bracket style, whitespace)...
image

**Comment by dkocich** [comment]
It would be maybe good to switch positions of "info" and "Open in" buttons and use drop-up (https://valor-software.com/ngx-bootstrap/#/dropdowns#dropup) instead of a drop-down menu. What do you think (those buttons are partially outside of a screen...)?
image

File: public_src/components/toolbar/toolbar.component.html [hunk 3]
```diff
@@ -54,24 +54,39 @@
...
+      &nbsp;{{ 'Open in ...' | translate }}
+      <span class="caret"></span>
+    </button>
+    <ul *dropdownMenu id="dropdownitems" class="dropdown-menu" role="menu">
+      <li role="menuitem">
+        <a class="dropdown-item" href="#" tooltip="{{ 'Open in ID' | translate }}" (click)="openInIdEditor(currentElement)">Open in ID
+        </a>
+      </li>
+      <li role="menuitem">
+        <a class="dropdown-item" href="#" tooltip="{{ 'Open JOSM and show selected element' | translate }}" (click)="getLoadAndZoomUrl()">Open in JOSM
```
[the remark this answers is not in the document]
Comment (anushasingh) [reply public_src/components/toolbar/toolbar.component.html]: Should I include <i class="fa fa-download" aria-hidden="true"></i> as well as  <span class="caret"></span> ?

[the remark this answers is not in the document]
Comment (anushasingh) [reply public_src/components/toolbar/toolbar.component.html]: I have included `[disabled]="isDisabled()"` in the main button.

[the remark this answers is not in the document]
Comment (anushasingh) [reply public_src/components/toolbar/toolbar.component.html]: Okay, I will include it for the JOSM and remove it from the main button.

[the remark this answers is not in the document]
Comment (anushasingh) [reply public_src/components/toolbar/toolbar.component.html]: I think it was okay before, as `[disabled]="isDisabled()"` should be in the main button.

File: public_src/components/toolbar/toolbar.component.less [hunk 4]
```diff
@@ -52,3 +52,11 @@
...
   padding: 10px;
 }
+#dropdown{
+  margin: 0;
+  padding: 0;
+}
+
+#dropdownitems{
+  width:100%;
+}
```
[the remark this answers is not in the document]
Comment (anushasingh) [reply public_src/components/toolbar/toolbar.component.less]: I added the line in my last two commits. I can't seem to understand why is it not showing up.

**Pull request merged**